spec/integration/data_stream/manifest.spec.yml

```diff
@@ -490,6 +490,16 @@ spec:
         dynamic_namespace:
           description: When set to true, agents running this integration are granted data stream privileges for all namespaces of its type
           type: boolean
+    owner:
```

Comment:
Please reuse definition from main manifest, something like:
```suggestion
    owner:
      $ref: "../../manifest.spec.yml#/definitions/onwer"
```

Or reuse only the `github` property if we don't want the `type`.